cpp/collections_module/algorithm/algorithm.hpp

```diff
@@ -0,0 +1,10 @@
+#include <mgp.hpp>
+#include <string>
+
+namespace Collections {
+
+constexpr std::string_view kReturnToSet = "result";
+constexpr std::string_view kProcedureToSet = "toSet";
```

Comment:
can we call this "to_set". I think this is the style we have been using for now in Memgraph procedures